aten/src/ATen/native/native_functions.yaml

```diff
@@ -835,6 +835,11 @@
     CPU: _frac_out_cpu
     CUDA: _frac_out_cuda
 
+- func: _from_blob(void* data, int[] sizes, int[] strides, std::function<void(void*)> deleter, *, ScalarType? dtype=None, Layout? layout=None, Device? device=None, bool? pin_memory=None) -> Tensor
```

Comment:
This doesn't seem like the best solution, because native_functions is supposed to basically be python and now you need to teach codegen about these crazy things like `std::function<void(void*)> deleter` just to get type-dispatch.

Can we just hardcode this function in your registration table, i.e. just make it the equivalent of type?  Because its' really just implementation detail stuff that doesn't need to be in native_functions (outside dispatch).